lib/python/src/bailo/helper/release.py

```diff
@@ -171,12 +227,26 @@ def upload(self, path: str, data: BytesIO | None = None) -> str:
                 name = path
 
             with open(path, "rb") as f:
-                res = self.client.simple_upload(self.model_id, name, f).json()
+                data = BytesIO(f.read())
```

Comment:
This won't work with large files, it loads the entire file contents into memory.

Reply:
Done. Added new integration test to prevent this in future